src/visualization/parsers/perceivedLoadTimeLayout.ts

```diff
@@ -0,0 +1,45 @@
+/// <reference path="../../../node_modules/clarity-js/clarity.d.ts" />
+import { IParser } from "../components/Snapshot";
+import { IAttributes, ILayoutState, IElementLayoutState, IDoctypeLayoutState, ITextLayoutState, IIgnoreLayoutState, Action } from "clarity-js/clarity";
+import { IPerceivedElementLayoutState } from "./IPerceivedElementLayoutState";
+import { Layout } from  "../parsers/layout";
+
+export default class PerceivedLoadTimeLayout extends Layout {    
+    
+    setup(document: Document, frame: HTMLIFrameElement, base: string, thumbnail? : boolean) {
+        this.layouts = {};
+        this.document = document;
+        this.base = base;
+    }
+
+    protected insertHelper(state: IPerceivedElementLayoutState, img : HTMLImageElement ){
+        if(state.isPerceivedEvent === false ){
+            img.style.color = "red";
```

Comment:
>img.style.color = "red"; [](start = 12, length = 24)

I've noticed the opacity, but haven't noticed the color change when I've seen the visualizations. Is changing the color doing anything? If so this is fine, but if not we should just remove it